config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -14,15 +14,13 @@ periodics:
   interval: 2h
   spec:
     containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201014-d8fda77-master
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201014-882eb3f-master
+        command:
+        - runner.sh
+        - kubetest
         args:
-        - --timeout=920
-        - --bare
-        - --scenario=kubernetes_e2e
-        - --
```

Comment:
This is missing flags to tell kubetest what to do, I would add these:
https://github.com/kubernetes/test-infra/blob/deb1bd1036360a6357f5d7948bc287301991c213/config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml#L73-L82

Reply:
--upgrade_arg will run the upgrade test.

`kubetest --extract=ci/latest --extract=ci/latest-1.19 --check-version-skew=false --check-leaked-resources --gcp-node-image=gci  --gcp-zone=us-west1-b --upgrade_args="--ginkgo.focus=\[Feature:BoundServiceAccountTokenVolume\] --upgrade-target=ci/latest --upgrade-image=gci"` works for me locally.